Lib/annotationlib.py

```diff
@@ -0,0 +1,635 @@
+"""Helpers for introspecting and wrapping annotations."""
+
+import ast
+import enum
+import functools
+import sys
+import types
+
+
+class Format(enum.IntEnum):
+    VALUE = 1
+    FORWARDREF = 2
+    SOURCE = 3
+
+
+_Union = None
+_sentinel = object()
+
+# Slots shared by ForwardRef and _Stringifier. The __forward__ names must be
+# preserved for compatibility with the old typing.ForwardRef class. The remaining
+# names are private.
+_slots = (
```

Comment:
```suggestion
_PRIVATE_SLOTS = (
```

maybe it's easier to spot that it's a constant that way

Reply:
Will make it uppercase, I don't think we need the `_PRIVATE` part.